src/main/java/core/basesyntax/dao/ReportDaoCsvImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportDaoCsvImpl implements ReportDao {
```

Comment:
same as with reader, but without logic mixing
```suggestion
public class CsvWriter implements Writer {
```

Reply:
ok